packages/itwinui-react/src/core/Tabs/Tabs.tsx

```diff
@@ -280,7 +291,7 @@ export const Tabs = (props: TabsProps) => {
     ownerDoc.addEventListener('wheel', enableHorizontalScroll);
   }, [overflowOptions?.useOverflow, orientation, enableHorizontalScroll]);
 
-  const isTabHidden = (activeTab: HTMLElement, isVertical: boolean) => {
+  const isTabHidden = (tab: HTMLElement, isVertical: boolean) => {
```

Comment:
nit: Might be helpful to add JSDocs for this. E.g. to explain the three return values.

Reply:
This was removed